nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -412,7 +412,7 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
   `wants`), because the dependency that `multi-user.target` has on
   `network-online.target` is planned for removal.
 
-- `services.pgbouncer` now has systemd support enabled and will log to journald. The default setting for `services.pgbouncer.logFile` is now `null` to disable logging to a separate log file.
+- `services.pgbouncer` now has systemd support enabled and will log to journald. The systemd service is configured with `Type=notify-reload` and allows reloading configuration without process restart. PgBouncer configuration options were moved to the free-form type option named [`services.pgbouncer.settings`](#opt-services.pgbouncer.settings) according to the NixOS RFC 0042.
```

Comment:
```suggestion
- `services.pgbouncer` now has systemd support enabled and will log to journald.
  The systemd service is configured with `Type=notify-reload` and allows reloading configuration without process restart.
  PgBouncer configuration options were moved to the free-form type option named [`services.pgbouncer.settings`](#opt-services.pgbouncer.settings) according to the NixOS RFC 0042.
```

Reply:
👍
Also we should probably wait for 24.11 release notes file to be initialized, see above w.r.t. release schedule.

Reply:
Moved to 24.11 release notes.